src/handle_action.cpp

```diff
@@ -966,22 +966,29 @@ static void sleep()
     }
 
     time_duration try_sleep_dur = 24_hours;
+    std::string deaf_text = "";
+    if( g->u.is_deaf() ) {
+        deaf_text = _( "<color_c_red> (DEAF!)</color>" );
+    } else {
+        deaf_text = "";
+    }
```

Comment:
This is useless. In all cases of non-deaf player `deaf_text` will be empty string anyway, as you set it on line 969.